rr_common/src/image_transformation/pointcloud_projector.cpp

```diff
@@ -68,10 +78,34 @@ class PointCloudProjector : public nodelet::Nodelet {
         pointcloud_pub_.publish(out);
     }
 
+    void axes_callback(const rr_msgs::axes& ax) {
+        if (!axes_init) {
+            // Set the initial axes
+            tf2::convert(cam_geom_.GetCameraPose().orientation, initial_axes);
+        }
```

Comment:
Does this assume that the car is upright and flat when the node is launched? That's fine as long as it's documented